packages/react-router/tests/Matches.test-d.tsx

```
@@ -1,4 +1,6 @@
 import { expectTypeOf, test } from 'vitest'
+import type * as React from 'react'
+import type { MakeMatchRouteOptions } from '../src'
```

ESLint

[error] '../src' imported multiple times.

(import/no-duplicates)